src/events/guildMemberRemove.ts

```diff
@@ -13,16 +13,28 @@ export default async function guildMemberRemoveHandler(member: GuildMember | Par
 	const user = member.user;
 	
 	const auditLogs = await guild.fetchAuditLogs({
-		limit: 1
+		limit: 10
```

Comment:
Why the increase?

Reply:
We can't limit the number of audit logs, we just filter them afterwards, if there was another most recent audit log here we'd miss it, so we may as well bring back some more. I don't have any evidence to show what a better number for this is

Reply:
That's fair. It's very very unlikely, but it's true that it's a race condition